### PR TITLE
Documentation: minor additional doc fixes

### PR DESCRIPTION
**Pull request by jrfnl**
Fixes a few parameter comments.

### CODE REVIEW

File: src/messages/Whip_InvalidVersionRequirementMessage.php [hunk 1]
```diff
@@ -16,15 +16,15 @@ class Whip_InvalidVersionRequirementMessage implements Whip_Message {
 	private $requirement;
 
 	/**
-	 * @var
+	 * @var int
```
Comment (moorscode) [inline src/messages/Whip_InvalidVersionRequirementMessage.php]: As this parameter is holding a `version` number (4.1.2), the type of the variable should be `string`

File: src/messages/Whip_InvalidVersionRequirementMessage.php [hunk 2]
```diff
@@ -16,15 +16,15 @@ class Whip_InvalidVersionRequirementMessage implements Whip_Message {
...
 	 */
 	private $detected;
 
 	/**
 	 * Whip_InvalidVersionRequirementMessage constructor.
 	 *
 	 * @param Whip_VersionRequirement $requirement
-	 * @param                         $detected
+	 * @param int                     $detected
```
Comment (moorscode) [inline src/messages/Whip_InvalidVersionRequirementMessage.php]: As this parameter is holding a `version` number (4.1.2), the type of the variable should be `string`

**Comment by jrfnl** [comment]
@moorscode Did you see my note about this in the commit message ?

> Based on the fact that this class is only called from the `Whip_RequirementsChecker::addMissingRequirementMessage()` method and in that function call the return value of a call to the `Whip_Configuration::configuredVersion()` method is passed and that the `Whip_Configuration::configuredVersion()` method according to its documentation returns an integer.

If it's `string`, existing documentation is wrong and needs to be fixed.

**Comment by jrfnl** [comment]
@moorscode OK, so I've dug a little deeper and agree with you that it can *also* be a string, so I have updated the two new parameter/property type comments, added a short description to both and I have adjusted the existing documentation on which I'd based the original values.

I've done this in a separate commit for easier review. Please let me know if you want me to squash this last commit into the previous one.

**Pull request merged**